api/src/test/java/org/openmrs/module/fhir2/api/impl/FhirTaskServiceImplTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.impl;
+
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.when;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.openmrs.module.fhir2.Task;
+import org.openmrs.module.fhir2.api.dao.FhirTaskDao;
+import org.openmrs.module.fhir2.api.translators.TaskTranslator;
+import org.openmrs.module.fhir2.api.translators.impl.TaskTranslatorImpl;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FhirTaskServiceImplTest {
+	
+	private static final String TASK_UUID = "dc9ce8be-3155-4adf-b28f-29436ec30a30";
+	
+	private static final org.hl7.fhir.r4.model.Task.TaskStatus FHIR_TASK_STATUS = org.hl7.fhir.r4.model.Task.TaskStatus.REQUESTED;
+	
+	private static final org.hl7.fhir.r4.model.Task.TaskStatus FHIR_NEW_TASK_STATUS = org.hl7.fhir.r4.model.Task.TaskStatus.ACCEPTED;
+	
+	private static final org.openmrs.module.fhir2.Task.TaskStatus OPENMRS_TASK_STATUS = org.openmrs.module.fhir2.Task.TaskStatus.REQUESTED;
+	
+	private static final org.openmrs.module.fhir2.Task.TaskStatus OPENMRS_NEW_TASK_STATUS = org.openmrs.module.fhir2.Task.TaskStatus.ACCEPTED;
+	
+	private static final org.hl7.fhir.r4.model.Task.TaskIntent FHIR_TASK_INTENT = org.hl7.fhir.r4.model.Task.TaskIntent.ORDER;
+	
+	private static final org.openmrs.module.fhir2.Task.TaskIntent OPENMRS_TASK_INTENT = org.openmrs.module.fhir2.Task.TaskIntent.ORDER;
+	
+	@Mock
+	FhirTaskDao dao;
+	
+	@Mock
+	TaskTranslator translator;
+	
+	private FhirTaskServiceImpl fhirTaskService;
+	
+	@Before
+	public void setUp() {
+		fhirTaskService = new FhirTaskServiceImpl();
+		fhirTaskService.setDao(dao);
+		fhirTaskService.setTranslator(translator);
+	}
+	
+	@Test
+	public void shouldGetTaskByUuid() {
```

Comment:
I think we'd prefer the "shouldRetrieveTaskByUuid" terminology to stay consistent with other tests.